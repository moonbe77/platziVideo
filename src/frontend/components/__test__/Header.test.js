import React from 'react';
import { create } from 'react-test-renderer';
import ProviderMock from '../../__mocks__/MockProvider';
import Header from '../Header';

describe('Header Component', () => {
  test('match Snapshot', () => {
    const header = create(
      <ProviderMock>
        <Header />
      </ProviderMock>,
    );
    expect(header.toJSON()).toMatchSnapshot();
  });
});
