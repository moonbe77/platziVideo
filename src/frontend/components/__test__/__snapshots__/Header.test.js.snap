// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component match Snapshot 1`] = `
<header
  className="header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Platzi Video"
      className="header__img"
      src="test-file-stubs"
    />
  </a>
  <a
    href="/create_movie"
    onClick={[Function]}
  >
     Create Movie
  </a>
  <a
    href="/player/5e9110ab7aae6f44e0b93a89"
    onClick={[Function]}
  >
     Player 
  </a>
  <div />
  <div
    className="header__menu"
  >
    <div
      className="header__menu--profile"
    >
      <img
        alt=""
        src="test-file-stubs"
      />
      <p>
        Perfil
      </p>
    </div>
    <ul>
      <li>
        <a
          href="/login"
          onClick={[Function]}
        >
          Iniciar sesi√≥n
        </a>
      </li>
    </ul>
  </div>
</header>
`;
